vendors/st/boards/stm32l475_discovery/aws_tests/STM32L475VGTx_FLASH.ld

```diff
@@ -43,8 +43,8 @@ MEMORY
 {
 RAM (xrw)      : ORIGIN = 0x20000000, LENGTH = 96K
 RAM2 (xrw)      : ORIGIN = 0x10000000, LENGTH = 32K
-FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 400K		/* Use only the first bank */
-FLASH_UC (r)	: ORIGIN = 0x08064000, LENGTH = 9K		/* Fixed-location area */
+FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 512K		/* Use only the first bank */
```

Comment:
Why is this change needed? The comment says, use only first bank and if  I remember correctly, size of each bank is 512K.

Reply:
I think 400K is enough for test application. This will be revert back.

Currently footprint for cellular
Cellular common : 41K
MbedTLS : 60K
Secure sockets cellular : 11K

Also the FLASH_UC is overlapped. It will be updated.
aws_demos
FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 503K    /* Use only the first bank */
FLASH_UC (r)	: ORIGIN = 0x0807DC00, LENGTH = 9K		/* Fixed-location area */

aws_tests:
FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 400K    /* Use only the first bank */
FLASH_UC (r)	: ORIGIN = 0x08064000, LENGTH = 9K		/* Fixed-location area */